app/views/arrival_details.scala.html

```diff
@@ -46,6 +46,8 @@
             hint = Some(messages("arrivalDetails.time.hint"))
         )
 
-        @components.submit_button()
+        @components.submit_button {
+            @messages("site.continue")
+        }
```

Comment:
Is it worth to split it to two lines? I think one will be better. Just style comment

Reply:
Could be - on the other hand if treated as HTML messages outcome is inside